packages/builder/src/utils/template.ts

```diff
@@ -48,6 +185,39 @@ export function template(str?: string, options?: TemplateOptions) {
   };
 }
 
+/**
+ * Executes a template string in a SES secure compartment.
+ * @param templateStr - The template string to evaluate
+ * @param context - The context object containing endowments
+ * @param importKey - The key to use for imports (defaults to 'ctx')
+ * @returns The evaluated result
+ */
+export function executeTemplate(templateStr: string, context: Record<string, any> = {}, importKey: 'ctx' | 'recorders') {
```

Comment:
we almost always use `ctx` for `importKey`, so we should set the default value to `importKey`

also, if we rename the inner `template` function, we can avoid having to make changes to many files, which simplifies the look of this PR overall